ldes-fragmentisers/ldes-fragmentisers-timebased-hierarchical/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/timebasedhierarchical/model/FragmentationTimestampTest.java

```diff
@@ -16,12 +16,15 @@
 
 class FragmentationTimestampTest {
 
-	private static final LocalDateTime TIME = LocalDateTime.of(2023, 1, 2, 6, 30, 40);
+	private static final LocalDateTime TIME6am = LocalDateTime.of(2023, 1, 2, 6, 30, 40);
+	private static final LocalDateTime TIME6pm = LocalDateTime.of(2023, 1, 2, 18, 30, 40);
 
 	@ParameterizedTest(name = "test asString for granularity {0}")
 	@ArgumentsSource(TimeArgumentProvider.class)
-	void when_AsStringForGranularity_Then_ReturnCorrectString(Granularity granularity, String expected) {
-		FragmentationTimestamp fragmentationTimestamp = new FragmentationTimestamp(TIME, granularity);
+	void when_AsStringForGranularity_Then_ReturnCorrectString(LocalDateTime time,
+															  Granularity granularity,
+															  String expected) {
```

Comment:
formatting?

Reply:
Just looks weird in github, is formatted under each other by intellij standard CTRL + ALT + L